build_scripts/build_usd.py

```diff
@@ -1024,6 +1024,41 @@ def InstallTBB_Windows(context, force, buildArgs):
         CopyFiles(context, "lib\\intel64\\vc14\\*.*", "lib")
         CopyDirectory(context, "include\\serial", "include\\serial")
         CopyDirectory(context, "include\\tbb", "include\\tbb")
+        CopyFiles(context, "cmake\\TBBConfig.cmake", "lib\\cmake\\TBB")
+        CopyFiles(context, "cmake\\TBBConfigVersion.cmake", "lib\\cmake\\TBB")
```

Comment:
nitpick but It might be nice to use `os.path.join` instead of `\\`

Reply:
In this particular case, it was just mirroring what was already there.  Sure, I could change it for this, but I think if we did that we'd have to change the original ones as well that are outside of this PR.